freq: 2
background_b: 200
background_g: 0
background_r: 125
